src/prepare.c

```diff
@@ -480,10 +480,12 @@ static zend_class_entry* pmmpthread_copy_entry(const pmmpthread_ident_t* source,
 
 	memcpy(&prepared->info.user, &candidate->info.user, sizeof(candidate->info.user));
 
+#if PHP_VERSION_ID < 80400
 	if ((PMMPTHREAD_ZG(options) & PMMPTHREAD_INHERIT_COMMENTS) &&
 	   (candidate->info.user.doc_comment)) {
 			prepared->info.user.doc_comment = pmmpthread_copy_string(candidate->info.user.doc_comment);
 		} else prepared->info.user.doc_comment = NULL;
+#endif
```

Comment:
This is not removed, it's moved to a different place in the struct to allow internal classes to use it.